news/2018-05-14-twc-finals-recap.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "osu!taiko World Cup - Finals Week 1 Recap!"
+date: 2018-05-14 00:00:00 +0000
+---
+The drum mashing of the Taiko World Cup 2018 has concluded while the fruit puns of the osu!catch World Cup will soon commence. The journey for the gold medal has left us with one winning team, but definitely not in the dominant fashion that was expected. 
```

Comment:
Might want to remove the reference to osu!catch because it's also mentioned at the end.